changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
+      "[Added] Offer option to choose file from one branch or the other when resolving merge conflicts - #9702",
+      "[Added] Add context menu for highlighted text in diff view - #5100. Thanks @HashimotoYT!",
+      "[Fixed] Retain default branch option in \"Create Branch\" dialog when user focuses away from Desktop - #9611",
+      "[Improved] Clean up copy for onboarding sign-in flow - #9715",
```

Comment:
I don't think this made it to prod?
```suggestion
```

Reply:
@billygriffin these changes are a delta from the current wording on prod, so we should include this note.